src/benchmarks/real-world/ImageSharp/TestUtilities/TestImages.cs

```diff
@@ -0,0 +1,682 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.Linq;
+
+// ReSharper disable InconsistentNaming
+// ReSharper disable MemberHidesStaticFromOuterClass
+namespace SixLabors.ImageSharp.Tests
+{
+    /// <summary>
+    /// Class that contains all the relative test image paths in the TestImages/Formats directory.
+    /// Use with <see cref="WithFileAttribute"/>, <see cref="WithFileCollectionAttribute"/>.
+    /// </summary>
+    public static class TestImages
```

Comment:
I would review this class and only keep the images which are actually used, deleting the rest of the image files. (I know it's a lot of chore work ...)